### PR TITLE
Add support subchunk version 9

### LINKED ISSUE #11: Unsupported subchunk version 9
Error
 Unsupported subchunk version 9

### PR DESCRIPTION
**Pull request by XetPy1030**

### CODE REVIEW

File: bedrock/bedrock.py [hunk 1]
```diff
@@ -216,10 +216,13 @@ def __init__(self, db, x, z, y):
       except KeyError:
         raise NotFoundError("Subchunk at {} {}/{} not found.".format(x, z, y))
       self.version, data = data[0], data[1:]
-      if self.version != 8:
+      if self.version not in [8, 9]:
         raise NotImplementedError("Unsupported subchunk version {} at {} {}/{}".format(self.version, x, z, y))
       numStorages, data = data[0], data[1:]
 
+      if self.version == 9:
+        self.y_db, data = data[0], data[1:]
```
Comment (BluCodeGH) [inline bedrock/bedrock.py:224]: You should set `self.y_db` to `None` if the version is not 9, better to have a consistent set of properties even if they may be None than make the client deal with catching exceptions.

File: bedrock/bedrock.py [hunk 2]
```diff
@@ -284,6 +287,9 @@ def save(self, db, force=False):
         for block in palette:
           data += nbt.encode(block)
 
+      if self.version == 9:
+        data = self.y_db + data
```
Comment (BluCodeGH) [inline bedrock/bedrock.py]: Have you tested this code? I don't think you can concatenate an integer and a bytes object.

**Review by BluCodeGH — CHANGES_REQUESTED** [review]
Looks like a simple fix thankfully, just a few comments.

File: bedrock/bedrock.py (hunk 2 above)
Comment (XetPy1030) [reply bedrock/bedrock.py]: Yes, it works

**Review by BluCodeGH — APPROVED** [review]

**Pull request merged**